napalm_logs/config/opengear/AUTORESPONSE.yml

```diff
@@ -0,0 +1,20 @@
+#<30>ztp-tankstelle_1-ts autoresponse 2018-Nov- 9 15:41:30 AR: name=DeviceBoot, index=2, ref=2, state=Triggered, event=config.ports.port2.serial.event.pattern2.rx, value=Booting...
+
+messages:
+  - error: AUTORESPONSE
```

Comment:
Can you please clarify what's this exactly?

Reply:
The opengear console server is able to detect patterns in the serial data it received from an attached device. These patterns trigger an event on the console server which then is logged via syslog. We use such a pattern to detect the boot of a Cisco switch.